packages/devtools_app/lib/src/http/http_request_data.dart

```diff
@@ -99,17 +69,29 @@ class HttpRequestData {
     return data;
   }
 
-  final int _timelineMicrosBase;
+  static const _connectionInfoKey = 'connectionInfo';
+  static const _contentTypeKey = 'content-type';
+  static const _cookieKey = 'cookie';
+  static const _errorKey = 'error';
+  static const _filterKey = 'filterKey';
+  static const _localPortKey = 'localPort';
+  static const _methodKey = 'method';
+  static const _requestHeadersKey = 'requestHeaders';
+  static const _responseHeadersKey = 'responseHeaders';
+  static const _statusCodeKey = 'statusCode';
+  static const _setCookieKey = 'set-cookie';
```

Comment:
make this key camelCase?

Reply:
perhaps add a TODO and file a bug upstream on the VM to cleanup the names.

Reply:
added a TODO and filed a bug. Names need to be changed upstream